huaweicloud/services/modelarts/resource_huaweicloud_modelarts_resource_pool.go

```diff
@@ -87,6 +116,10 @@ func ResourceModelartsResourcePool() *schema.Resource {
 				Computed:    true,
 				Description: `The description of the resource pool.`,
 			},
+			"charging_mode": common.SchemaChargingMode(nil),
+			"period_unit":   common.SchemaPeriodUnit(nil),
+			"period":        common.SchemaPeriod(nil),
+			"auto_renew":    common.SchemaAutoRenewUpdatable(nil),
```

Comment:
suggest to add `status` and `resource_pool_id` attributes

Reply:
done